examples/renv-shiny/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM rhub/r-minimal:4.2.3-patched
+
+COPY installr /usr/local/bin/
+
+RUN wget https://github.com/jgm/pandoc/releases/download/2.13/pandoc-2.13-linux-amd64.tar.gz && \
```

Comment:
We could install a newer pandoc version, that's pretty old.

Also, we could use `ARG TARGETPLATFORM` and then `${TARGETPLATFORM}`, to make this work on arm64 as well.

Reply:
Changed to a new version of pandoc.